Services/Init/classes/class.ilStartUpGUI.php

```diff
@@ -790,26 +790,17 @@ protected function showLoginForm(string $page_editor_html, ilPropertyFormGUI $fo
     {
         global $tpl;
 
-        // @todo move this to auth utils.
-        // login via ILIAS (this also includes ldap)
-        // If local authentication is enabled for shibboleth users, we
```

Comment:
The comment said: 

```
If local authentication is enabled for shibboleth users, we display the login form for ILIAS here.
```

But exactly this did **not** happen. If CAS or Shibboleth is set as the default authentication method, the standard login form is neither built nor rendered.